README.md

```diff
@@ -1,4 +1,10 @@
-# Ol-Explorer
+# Ol-Explorer - ARCHIVED
+
+You can run this webapp locally on your computer by following the
+instructions from the [ByzCoin](https://github.com/c4dt/byzcoin)
+project.
+
```

Comment:
This was not very clear to me as someone who doesn't know the project, how about this:

> This project is intended to be run together with a blockchain such as ByzCoin. To test it locally, we recommend that you follow the instructions for setting up the [ByzCoin](https://github.com/c4dt/byzcoin) project, and choose "Go to the OL Explorer" in the application menu.